clustergroup/templates/_helpers.tpl

```diff
@@ -202,13 +202,19 @@ kind: OperatorGroup
 metadata:
   name: {{ $k }}-operator-group
   namespace: {{ $k }}
+      {{- if (hasKey $v "targetNamespaces") }}
+      {{- if $v.targetNamespaces }}
+      {{- if (len $v.targetNamespaces) }}
 spec:
   targetNamespaces:
-      {{- if (hasKey $v "targetNamespaces") }}
         {{- range $v.targetNamespaces }}{{- /* We loop through the list of tergetnamespaces */}}
   - {{ . }}
         {{- end }}{{- /* End range targetNamespaces */}}
```

Comment:
Can we fix the comments here? It's a bit confusing to understand which end refers to what

Reply:
Maybe we can also indent them to the left a bit (the if statements and the end statements) as it makes them a bit more readable

Reply:
@mbaldessari I will clean this up.